assets/js/components/notifications/GA4AdSenseLinkedNotification.js

```diff
@@ -225,3 +216,8 @@ export default function GA4AdSenseLinkedNotification() {
 		</Grid>
 	);
 }
+
+export default whenInViewContext( { allNonViewOnly: true } )(
+	// eslint-disable-next-line sitekit/acronym-case
```

Comment:
That's odd, this component name didn't flag errors before, strange this is needed 🤔 

Yeah, just checked, that's definitely a false positive. I've filed #9048.